cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -99,8 +99,8 @@ func TestEvaluateExecute(t *testing.T) {
 	type testCase struct {
 		Name string
 
-		Before func(t *testing.T, logger *log.Logger, resultPath string)
-		After  func(t *testing.T, logger *log.Logger, resultPath string)
+		Before func(t *testing.T, logger *log.Logger, resultPath string, testdataPath string)
```

Comment:
Dropping this commit. This is not a refactoring. And we should never modify the testdata repository. The command should make sure that the repository is copied, not we in our tests.